Makefile

```diff
@@ -458,18 +458,34 @@ dev-release:
 	$(Q)kubectl apply -f ./test/operator-hub/operator_source.yaml
 	# Subscribing to the operator
 	$(Q)kubectl apply -f ./test/operator-hub/subscription.yaml
-	while [ "$(kubectl get pods --field-selector=status.phase=Running -n openshift-marketplace" == "No resources found." ]
-	do
-		sleep 1s
-	done
+	#$(Q)kubectl get pods -n openshift-marketplace | grep "example-operators" | awk '{print $3}'
+	@if [ "kubectl get pods -n openshift-marketplace | grep "example-operators" | awk '{print $3}'" == "Running"] ; \
+	then \
+			echo "Operator marketplace pod is running"; \
+	fi
+	$(Q)$(HACK_DIR)/check-crds.sh
+	INSTALL_PLAN_PRIOR ?= service-binding-operator.v${BUNDLE_VERSION}
 	$(eval VERSION_NUMBER := $(shell kubectl get csvs  -n=default -o jsonpath='{.items[*].spec.version}'))
 	@if [ ${VERSION_NUMBER} == ${BUNDLE_VERSION} ]; \
 	then \
-    	echo -e "OLM Bundle Version validation succeeded \n " \
+    	echo -e "OLM Bundle Version validation succeeded \n ";
```

Comment:
IMHO, it would be better for readability and maintainability to extract `dev-release` target into to a standalone script and invoke it from `make`.

Reply:
Agreed